core/identity/id.go

```diff
@@ -41,19 +42,19 @@ func IDFromMarshalUtil(marshalUtil *marshalutil.MarshalUtil) (id ID, err error)
 	return
 }
 
+// IDFromBytes decodes ID from bytes.
+func IDFromBytes(bytes []byte) (id ID, err error) {
+	if _, err := serix.DefaultAPI.Decode(context.Background(), bytes, &id); err != nil {
+		return ID{}, errors.Wrap(err, "failed to decode node identity from bytes")
```

Comment:
is the error message "node" identity correct here? Or is it just an identity?

Reply:
Amended